### PR TITLE
Merchant finder

### PR DESCRIPTION
**Pull request by blsrofe**
Merchant tests are passing for show and index. The tests are not passing for find and find_all. There is something wrong in the controller that I am not sure how to fix yet.

### CODE REVIEW

**Comment by blsrofe** [comment]
Spec harness is passing now.

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,27 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  namespace :api do
```
Comment (erose357) [inline config/routes.rb]: I recommend DRYing up the routes so the name spacing isn't repeated so much.  This is how I started invoices:
  namespace :api do
    namespace :v1 do
      namespace :invoices do
        get '/find', to: 'search#show'
      end
      resources :merchants, only: [:index, :show]
      resources :invoices, only: [:index, :show]
    end
  end
end

**Review by erose357 — CHANGES_REQUESTED** [review]
I recommend cleaning up the routes file.  Everything else looks great.  Nice job!

**Comment by blsrofe** [comment]
@erose357 The routes are fixed and invoices are merged in. We will have to figure out which version of serializers we want to use. I'm not sure if it matters or not.

**Review by erose357 — APPROVED** [review]

**Pull request merged**